### PR TITLE
Update migration guide and add WIP label

### PR DESCRIPTION
**Pull request by YvonneW**
The following files were updated to improve structure subsequent to HTML-to-markdown conversion:
- en/docs/setup/migrating-data-publishers.md
- en/docs/setup/migration-guide.md
- en/docs/setup/understanding-what-has-changed.md
- en/docs/setup/upgrading-from-an-older-version-of-wso2-is.md
- en/docs/setup/upgrading-from-the-previous-release.md
- en/mkdocs.yml

The following files were updated add the WIPlabel in the header: 
- /en/docs/assets/css/theme.css
- /en/theme/material/partials/header.html

Created the following folder: 
- /en/docs/assets/img/setup

Added the following files: 
- /en/docs/assets/img/setup/new-design-approach.png
- /en/docs/assets/img/setup/old-design-approach.png
- /en/docs/assets/attachments/wso2is-5.8.0-migration.zip

### CODE REVIEW

File: en/docs/assets/css/theme.css [hunk 1]
```diff
@@ -821,3 +821,11 @@ html .md-typeset .superfences-tabs > label:hover {
...
     width: 12.1rem;
 }
+
+.beta-badge {
+    display: inline-block;
+    background: #ffd600;
+    font-size: 14px;
+    padding: 5px;
+    line-height: 1;
+}
```
[the remark this answers is not in the document]
Comment (YvonneW) [reply en/docs/assets/css/theme.css]: If we change it static, then the WIP label will be associated with the corresponding page title as well. So, the current behavior is correct.

File: en/docs/setup/upgrading-from-the-previous-release.md [hunk 2]
```diff
@@ -264,12 +240,11 @@ Follow the steps below to perform the upgrade.
...
               MANAGED BY AUTOMATIC STORAGE USING STOGROUP IBMSTOGROUP
               EXTENTSIZE 4;
-    ```
-
+        ```
 
 3.  Do the following database updates:  
     1.  Download the [migration
-        resources](https://docs.wso2.com/download/attachments/56986329/wso2is-5.8.0-migration.zip?version=1&modificationDate=1557719865000&api=v2)
+        resources](../../assets/attachments/wso2is-5.8.0-migration.zip?)
```
[the remark this answers is not in the document]
Comment (YvonneW) [reply en/docs/setup/upgrading-from-the-previous-release.md]: Done

File: en/docs/setup/upgrading-from-the-previous-release.md [hunk 3]
```diff
@@ -156,7 +139,7 @@ production environment.
         copy of the `               conf              ` directory
         according to the features that you require . For a detailed list
         of configuration changes, see [Configuration
-        changes](Understanding-What-Has-Changed_119128182.html#UnderstandingWhatHasChanged-Configurationchanges)
+        changes](../../setup/understanding-what-has-changed#UnderstandingWhatHasChanged-Configurationchanges)
```
[the remark this answers is not in the document]
Comment (YvonneW) [reply en/docs/setup/upgrading-from-the-previous-release.md]: Done

[the remark this answers is not in the document]
Comment (sherenewso2) [reply en/docs/setup/upgrading-from-the-previous-release.md]: The change doesn't seem to have got committed. Could you check again please?

File: en/docs/setup/upgrading-from-the-previous-release.md [hunk 4]
```diff
@@ -53,60 +40,56 @@ Follow this guide before you begin migration.
     5.7.0. This backup is necessary in case the migration causes issues
     in the existing database.
 4.  We recommend running the [cleanup
-    scripts](https://docs.wso2.com/display/IS580/Removing+Unused+Tokens+from+the+Database)
+    scripts](../../using-wso2-identity-server/removing-unused+Tokens+from+the+Database)
```
[the remark this answers is not in the document]
Comment (sherenewso2) [reply en/docs/setup/upgrading-from-the-previous-release.md]: @YvonneW

[the remark this answers is not in the document]
Comment (YvonneW) [reply en/docs/setup/upgrading-from-the-previous-release.md]: Made the changes manually.

**Pull request merged**